app/models/transaction.rb

```diff
@@ -28,7 +28,7 @@ class Transaction < ActiveRecord::Base
   default_scope { order(created_at: :desc) }
   scope :by_currency, ->(currency) { joins(:bank_account).where('bank_accounts.currency' => currency) }
 
-  validates :amount, presence: true
+  validates :amount, presence: true, length: { maximum: 255 }
```

Comment:
@Gataniel I suggest to use lower max_length, cos it's a integer in db... Let's use `length: { maximum: 10 }`